scipy/stats/tests/test_stats.py

```diff
@@ -1890,28 +1897,28 @@ def test_mad_empty(self):
         assert_equal(mad, np.nan)
 
     def test_mad_nan_propagate(self):
-        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
-                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
-                3.7, 3.7,3.77, 5.28, np.nan])
-
-        mad = stats.median_absolute_deviation(dat, nan_policy='propagate')
+        mad = stats.median_absolute_deviation(self.dat_nan, nan_policy='propagate')
         assert_equal(mad, np.nan)
 
     def test_mad_nan_raise(self):
-        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
-                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
-                3.7, 3.7,3.77, 5.28, np.nan])
-
         with assert_raises(ValueError):
-            stats.median_absolute_deviation(dat, nan_policy='raise')
+            stats.median_absolute_deviation(self.dat_nan, nan_policy='raise')
 
     def test_mad_nan_omit(self):
-        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
-                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
-                3.7, 3.7,3.77, 5.28, np.nan])
-
-        mad = stats.median_absolute_deviation(dat, nan_policy='omit')
-        assert_almost_equal(mad, 0.504084)
+        mad = stats.median_absolute_deviation(self.dat_nan, nan_policy='omit')
+        assert_almost_equal(mad, 0.34)
+
+    def test_mad_scale_default(self):
+        mad = stats.median_absolute_deviation(self.dat, scale="raw")
+        mad_float = stats.median_absolute_deviation(self.dat, scale=1.0)
+        assert_almost_equal(mad, 0.355)
+        assert_almost_equal(mad, mad_float)
+
+    def test_mad_scale_default(self):
+        mad = stats.median_absolute_deviation(self.dat, scale="normal")
+        mad_float = stats.median_absolute_deviation(self.dat, scale=1/1.4826022185056018)
```

Comment:
Here `0.67448975= 1/1.4826022185056018`, I've written this way (as 1/1.4826) to provide a link between what is commonly written (e.g. on wikipedia) and how the code does the calculation/division.